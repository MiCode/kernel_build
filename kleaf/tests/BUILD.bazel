# Copyright (C) 2022 The Android Open Source Project
#
# Licensed under the Apache License, Version 2.0 (the "License");
# you may not use this file except in compliance with the License.
# You may obtain a copy of the License at
#
#       http://www.apache.org/licenses/LICENSE-2.0
#
# Unless required by applicable law or agreed to in writing, software
# distributed under the License is distributed on an "AS IS" BASIS,
# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
# See the License for the specific language governing permissions and
# limitations under the License.

load("@bazel_skylib//:bzl_library.bzl", "bzl_library")

test_suite(
    name = "tests",
    tests = [
        "//build/kernel/kleaf/tests/cc_testing",
        "//build/kernel/kleaf/tests/ddk_test",
        "//build/kernel/kleaf/tests/debug_test",
        "//build/kernel/kleaf/tests/hermetic_test",
        "//build/kernel/kleaf/tests/kernel_build_config_test",
        "//build/kernel/kleaf/tests/kernel_build_test",
        "//build/kernel/kleaf/tests/kernel_config_test",
        "//build/kernel/kleaf/tests/kernel_images_test",
        "//build/kernel/kleaf/tests/kernel_module_group_test",
        "//build/kernel/kleaf/tests/merged_kernel_uapi_headers_test",
        "//build/kernel/kleaf/tests/trim_test",
    ],
)

bzl_library(
    name = "empty_test",
    srcs = [
        "empty_test.bzl",
    ],
    visibility = ["//build/kernel/kleaf:__subpackages__"],
)
