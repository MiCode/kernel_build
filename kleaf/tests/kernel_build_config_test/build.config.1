BUILD_CONFIG_1=1
