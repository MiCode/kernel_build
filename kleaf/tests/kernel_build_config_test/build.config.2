BUILD_CONFIG_2=1
