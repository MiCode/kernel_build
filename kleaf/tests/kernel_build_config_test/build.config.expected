BUILD_CONFIG_2=1
BUILD_CONFIG_1=1
